DOCS/man/options.rst

```diff
@@ -4672,27 +4671,40 @@ The following video options are currently all specific to ``--vo=opengl`` and
         performance problems), and is for doing experiments only. Will not
         be used automatically.
 
+``--gpu-api=<type>``
+    Controls which type of graphics APIs will be accepted:
+
+    auto
+        Use any available API (default)
+    opengl
+        Allow only OpenGL (requires OpenGL 2.1+ or GLES 2.0+)
+
 ``--opengl-es=<mode>``
-    Select whether to use GLES:
+    Controls which type of OpenGL context will be accepted:
 
+    auto
+        Allow all types of OpenGL (default)
     yes
-        Try to prefer ES over Desktop GL
-    force2
-        Try to request a ES 2.0 context (the driver might ignore this)
+        Only allow GLES
     no
-        Try to prefer desktop GL over ES
-    auto
-        Use the default for each backend (default)
+        Only allow desktop/core GL
 
-``--opengl-fbo-format=<fmt>``
+``--opengl-restrict=<version>``
+    Restricts all OpenGL versions above a certain version. Versions are encoded
+    in hundreds, i.e. OpenGL 4.5 -> 450. As an example, --opengl-restrict=300
+    would restrict OpenGL 3.0 and higher, effectively only allowing 2.x
+    contexts. Note that this only imposes a limit on context creation APIs, the
+    actual OpenGL context may still have a higher OpenGL version. (Default: 0)
+
+``--fbo-format=<fmt>``
     Selects the internal format of textures used for FBOs. The format can
     influence performance and quality of the video output. ``fmt`` can be one
     of: rgb8, rgb10, rgb10_a2, rgb16, rgb16f, rgb32f, rgba12, rgba16, rgba16f,
     rgba32f. Default: ``auto``, which maps to rgba16 on desktop GL, and rgba16f
     or rgb10_a2 on GLES (e.g. ANGLE), unless GL_EXT_texture_norm16 is
     available.
 
-``--opengl-gamma=<0.1..2.0>``
+``--gpu-gamma=<0.1..2.0>``
```

Comment:
I feel like this makes little sense. It only needs to distinguish itself from `--gamma`, so maybe `--gamma-factor` or something?

Reply:
Good point. I like that name better